plugins/source/aws/docs/tables/aws_support_case_communications.md

```diff
@@ -4,7 +4,7 @@ This table shows data for Support Case Communications.
 
 https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeCommunications.html
 
-The primary key for this table is **_cq_id**.
+The composite primary key for this table is (**case_id**, **submitted_by**, **time_created**).
```

Comment:
Assuming this is good enough, having both the timestamp and author on the PK. `case_id` is from the parent.